extensions/ql-vscode/src/vscode-tests/no-workspace/sarifParser.test.ts

```diff
@@ -1,6 +1,6 @@
 import * as path from 'path';
 import * as chai from 'chai';
-import chaiAsPromised from 'chai-as-promised';
+import * as chaiAsPromised from 'chai-as-promised';
```

Comment:
I'm pretty sure this line and line 7 are not actually needed since `chaiAsPromised` is configured in `extensions/ql-vscode/src/vscode-tests/no-workspace/index.ts`.